src/node_http_common-inl.h

```diff
@@ -4,7 +4,6 @@
 #include "node_http_common.h"
 #include "node.h"
 #include "node_mem-inl.h"
-#include "env-inl.h"
```

Comment:
If removing this works, then it’s by accident – this file does require `env-inl.h` in order to work. Please leave the include here.

(You can test this by including only `node_http_common-inl.h` from a `.cc` file – compiling that file will fail without this line.)

Reply:
I can guarantee that removing this would break on some of the platforms as I had omitted this originally and had to fix it.

Reply:
Great thanks - any tips on better build testing? I git grep'ing for these items in header files and running `make -j8 test` as a blanket test. I see the x-platform integration tests pass...

@jasnell Please let me know if I'm stomping on anything you're doing. Just looking for some fun side-hacking :-)

Reply:
Nope you're good :-) ... I'll adjust on my end if there are any merge conflicts.